packages/fluttium/lib/src/actions/long_press_on.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/gestures.dart';
+import 'package:fluttium/fluttium.dart';
+
+/// {@template long_press_on}
+/// Long press on a node that matches the arguments.
+///
+/// This action can be invoked either using the short-hand version:
+///
+/// ```yaml
+/// - longPressOn: "Hello World"
+/// ```
+///
+/// Or using the verbose version:
+///
+/// ```yaml
+/// - longPressOn:
+///     text: "Hello World"
+/// - longPressOn:
+///     offset: [0.5, 0.5]
+/// ```
+/// {@endtemplate}
+class LongPressOn extends Action {
+  /// {@macro long_press_on}
+  const LongPressOn({
+    this.text,
+    this.offset,
+  });
+
+  /// Optional text to search for.
+  final String? text;
+
+  /// Optional offset to tap on.
+  final Offset? offset;
+
+  static int _pointerId = 0;
+
+  @override
+  Future<bool> execute(Tester tester) async {
+    final Offset center;
+    if (text != null) {
+      final node = await tester.find(text!);
+      if (node == null) {
+        return false;
+      }
+      center = node.center;
+    } else if (offset != null) {
+      center = offset!;
+    } else {
+      return false;
+    }
+
+    final pointer = _pointerId++;
+    tester.emitPointerEvent(
+      PointerDownEvent(pointer: pointer, position: center),
+    );
+    await tester.pump(duration: kLongPressTimeout + kPressTimeout);
```

Comment:
Just out of curiosity why are we adding the long press timeout with the press timeout instead of just using the long press timeout here?

Reply:
Because that is what Flutter does in the `WidgetTester`: https://api.flutter.dev/flutter/flutter_test/WidgetController/longPressAt.html

Reply:
Interesting...learned something new today 💡 
Naming-wise that's a bit confusing but makes sense to be consistent with Flutter 👍